src/resources/SchemaService/SchemaServiceInterfaces.ts

```diff
@@ -5,22 +5,18 @@ export interface SchemaEntities {
 }
 
 export interface SchemaEntity {
-    name?: string;
-    displayName?: string;
```

Comment:
Those changes to the interface are breaking changes, please make sure that your [commit message is formatted accordingly](https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-description-and-breaking-change-footer)

Reply:
will do

Reply:
done !